src/main/java/com/ratifire/devrate/dto/InterviewRejectedDto.java

```diff
@@ -0,0 +1,19 @@
+package com.ratifire.devrate.dto;
+
+import java.time.ZonedDateTime;
+import lombok.Builder;
+import lombok.Getter;
+
+/**
+ * DTO that represents rejection data needed to notify users that an interview was rejected.
+ */
+@Builder
+@Getter
+public class InterviewRejectedDto {
```

Comment:
Do we really need this DTO? Is it reusable? It looks like a specific data structure for a single task that serves only one purpose.